docs/host/host.md

```diff
@@ -251,3 +251,32 @@ When Ksmtuned starts, initialize `pages_to_scan` in KSM to 1000 (**Minimum Pages
 KSM starts when the available memory falls below the **Threshold Coefficient**. If it detects that it is running, `pages_to_scan` increments by 300 (**Boost**) every minute until it reaches 5000 (**Maximum Pages**).
 
 KSM will stop when the available memory is above the **Threshold Coefficient**. If it detects that it is stopped, `pages_to_scan` decrements by 100 (**Decay**) every minute until it reaches 1000 (**Minimum Pages**).
+
+## NTP Configuration
+
+Time synchronization is important for the distributed cluster architecture, so there is a simple way to configure the NTP servers on Harvester.
+
+In the previous version, Harvester only supported NTP configuration [during the installation process](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, it was necessary to manually update the configuration file on each node.
+
+In v1.2.0, Harvester supports NTP configuration in Harvester/Settings. You can configure NTP servers any time for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.
+
+![](/img/v1.3/host/harvester-ntp-settings.png)
+
+You could setup multiple NTP Servers like the following figure:
+
+![](/img/v1.3/host/harvester-ntp-settings-multiple.png)
+
+You can check the configuration result from the `node.harvesterhci.io/ntp-service` annotation in Kubernetes Nodes: 
+- `ntpSyncStatus`: synced or unsynced
+- `currentNtpServers`: current NTP servers
```

Comment:
```suggestion
You can check the settings in the `node.harvesterhci.io/ntp-service` annotation in Kubernetes nodes: 
- `ntpSyncStatus`: Status of the connection to NTP servers (possible values: `synced` and `unsynced`)
- `currentNtpServers`: List of existing NTP servers
```

Reply:
Please check if my understanding is correct. I could not find information on the annotation mentioned here.